modules/ROOT/pages/custom-policy-uploading-to-exchange.adoc

```diff
@@ -30,12 +33,77 @@ Only, configuring the correct credentials is left to do. Update the settings.xml
   </servers>
 </settings>
 ----
-
-After that, run `mvn deploy` to publish the policy to Exchange.
+. Run `mvn deploy` to publish the policy to Exchange.
 
 The custom policy is now available for you to apply to APIs that belong to the specified organization.
 
+== Deploying a Policy Created Without Using the Maven Archetype
+
+If you chose not to create your custom policy project using the provided Maven archetypes, you need to manually set up your Exchange deployment configurations:
+
+. Go to your pom.xml file and define your Exchange url as a property:
++
+[source,XML,linenums]
+--
+<properties>
+  <exchange.url>https://maven.anypoint.mulesoft.com/api/v1/organizations/{OrgId}/maven</exchange.url>
+</properties>
+--
+* Replace the {orgId} by using your organization's ID:
+** Get your organization ID from *Access Management > Organization*:
+*** Click the name of your organization.
+*** Copy the UUID from the browser address. For example, copy _2a4b93c3-7899-4ea7-9374-f787744d8784_ from the URL.
+. Additonally, in your pom.xml file, you need to define Exchange as a repository:
++
+[source,XML,linenums]
+--
+<repository>
+	<id>exchange-server</id>
+	<name>MuleSoft Exchange Environment</name>
+	<url>${exchange.url}</url>
+	<snapshots>
+		<enabled>true</enabled>
+	</snapshots>
+</repository>
+--
++
+And also as a distribution management:
++
+[source,XML,linenums]
+--
+<distributionManagement>
+    <repository>
+        <id>exchange-server</id>
+        <name>Corporate Repository</name>
+        <url>${exchange.url}</url>
+        <layout>default</layout>
+    </repository>
+</distributionManagement>
+--
+. Finally, define your Exchange credentials in your settings.xml file in your Maven .m2 directory:
```

Comment:
`settings.xml` `.m2` (and you can say "the" instead of "your" before settings.xml to reduce repetition.